### ISSUE TITLE
go to PyCon 2015!

### ISSUE THREAD

**Issue by whit537** [gratipay/inside.gratipay.com#111]
Per private email w/ @dianaclarke, I've submitted a sponsorship application at the open-source/community level on [the website](https://us.pycon.org/2015/).

**Comment by whit537** [comment]
I'm unable to make it to PyCon this year (there's a big retirement party for my dad on Friday the 10th). @chrisdev will be there and is willing to staff the booth. He would like to find someone else to join him. Maybe @lyndsysimon, and/or @aisipos?

We get one free registration with the booth. Maybe we can transfer that. Who wants it?

**Comment by dianaclarke** [comment]
@whit537 Your sponsorship comes with two full tickets to PyCon, not just one. And we're sold out, so those tickets are in hot demand.

**Comment by dianaclarke** [comment]
@whit537 PS. The busy times are listed here:

    https://us.pycon.org/2015/sponsors/exhibitfloor/

It's really important to have your booth staffed during those times.

Thanks!

**Comment by chrisdev** [comment]
Is there no one who wants to go to PyCon 2015? 
Free ticket and you can share my hotel room which will reduce the cost even more!
From my experience its hard to do an effective job at the boot with only one person.

**Comment by whit537** [comment]
Thanks for the info, @dianaclarke. Good news about two registrations! Shall I email Ewa to transfer one to @chrisdev? I've emailed @aisipos to see if he can use the other to help Chris with the Gratipay booth.

**Comment by dianaclarke** [comment]
@whit537 Please email pycon8-reg@cteusa.com to transfer your ticket.

**Comment by whit537** [comment]
@dianaclarke Will do, thanks! :-)

**Comment by whit537** [comment]
Registration transferred to @chrisdev! :dancer:

**Comment by chrisdev** [comment]
:+1:  @whit537 @dianaclarke

**Comment by lyndsysimon** [comment]
I'll be at PyCon this year, but will be representing COS and won't be available to staff the Gratipay booth.

I will bring decals, though!

**Comment by citruspi** [comment]
Hey,

I'd be interested in helping out with staffing the Gratipay booth. How would we plan who staffs when?

Because I'm happy to staff the booth but there are some talks/events that I'd prefer not to miss...

&mdash; @citruspi

**Comment by chrisdev** [comment]
Great! We can come up with a duty roster since PyCon is being so strict about boot staffing!
@citruspi is I assuming you already have your PyCon ticket?
@whit537 can we help him out?

**Comment by citruspi** [comment]
@chrisdev,

I do already have PyCon ticket.

&mdash; @citruspi

**Comment by whit537** [comment]
Thanks, @lyndsysimon! I believe COS and Gratipay are next door to one another! :-)

@citruspi Huzzah! Thank you! I'll email the PyCon registration folks. What's a good email address to cc: you (ping me at chad@zetaweb.com to if you want to keep it private)?

**Comment by citruspi** [comment]
@whit537,

The email address to cc: me with would be hello@mihirsingh.com.

&mdash; @citruspi

**Comment by chrisdev** [comment]
With no @whit537 we probably need some actual swag to spice up the boot

- [ ] Penny Punch or at least some pre punched pennies - you can send some prepunched pennies to @citruspi via UPS or something?
- [ ] The giant heart that @whit537 made for the parade. Is it too big?
- [ ] Posters?
- [ ] Tee shirts - I can bring about 20 with me. @whit537 do we not have a design worked out already? 
- [ ] Decals/stickers @lyndsysimon

**Comment by citruspi** [comment]
@chrisdev,

Are the t-shirts for volunteers (us) or for attendees? If it's for attendees, which I feel like it is, I think we'll want more than 20. If the hotel is fine with it, I'd be interested in finding out how much it would cost to ship fifty to a hundred to the hotel before we arrive.

&mdash; @citruspi

**Comment by whit537** [comment]
@citruspi @chrisdev Let's say your budget is $500. In the past we've spent this on electric + table and chairs for the booth. If you'd rather spend it on swag + shipping that's fine with me.

**Comment by chrisdev** [comment]
@whit537 So we have to pay for the tables and chairs??

**Comment by citruspi** [comment]
@whit537,

What exactly do you mean by "electric?" Are we paying for the electricity?

&mdash; @citruspi

**Comment by whit537** [comment]
We get a folding table and two chairs gratis. In the past we've added a "bistro table" and additional chairs. Electric service in the booth is also extra, yes.

**Comment by whit537** [comment]
I can ship the penny punch and the big heart coin if you like. The heart coin might need a little repair first.

Here's the form for ordering furniture and electric service:

https://ordering.ges.com/CA-00015D54/downloadForms

**Comment by chrisdev** [comment]
@whit537  where can we find out more about this? I'm not seeing the details in https://us.pycon.org/2015/sponsors/exhibitfloor/

**Comment by whit537** [comment]
Actually, that's the whole exhibitor's manual.

@chrisdev @citruspi Maybe we should have a call to discuss? There was a bit of a learning curve for me on this and I'd be happy to pass on my knowledge. :)

**Comment by chrisdev** [comment]
Yea I've looked at those forms :-1: 
So at @citruspi I think we need to do a Google hangout with @whit537 to bone up on this expo stuff
What times/date is good for you guys

**Comment by whit537** [comment]
@citruspi I sent an email asking to add you to the Gratipay registration.

**Comment by whit537** [comment]
Re: availability for a call, I will know more in a couple hours. I may go to NYC later this week ...

**Comment by citruspi** [comment]
@whit537,

Thanks, got it.

&mdash; @citruspi

**Comment by citruspi** [comment]
@whit537 @chrisdev,

Preferably not this week - I'll be traveling to Rochester for most of it.

&mdash; @citruspi

**Comment by chrisdev** [comment]
@whit537 @citruspi my preferred time to chat is  Saturday/Sunday 14/15th at 5.30 pm to 8.30pm GMT -4
In not convenient for you guys we can try for a Weekday  from around 6.30pm
I'm also up early so we can do always do something from 3.30am any day of the week

**Comment by whit537** [comment]
Thanks @chrisdev.

@citruspi Would Sunday the 15th at 7:00pm GMT-4 (US/Eastern) work for you?

**Comment by citruspi** [comment]
@whit537,

Yup, Sunday the 15th at 1900 GMT-4 definitely works for me.

&mdash; @citruspi

**Comment by chrisdev** [comment]
@whit537 @citruspi Google Hangout or Skype?

**Comment by citruspi** [comment]
@chrisdev,

I'd really prefer it not be Skype. What about [Firefox Hello](https://www.mozilla.org/en-US/firefox/hello/)?

&mdash; @citruspi

**Comment by chrisdev** [comment]
@citruspi I hate Skype but...
Firefox hello looks cool maybe we should give it a try? Saves me from having to reinstall Chrome on my machine

**Comment by citruspi** [comment]
@chrisdev,

Exactly! I don't want to have to install Skype and the only reason I use Google Chrome is for Hangouts, and when I use it my computer becomes very hot.

Plus, Firefox Hello is [open source](https://github.com/mozilla-services/loop-server) :wink: and only one of us is required to have Firefox to initiate the call. The rest of us can use any browser that supports WebRTC.

&mdash; @citruspi

**Comment by whit537** [comment]
> Yup, Sunday the 15th at 1900 GMT-4 definitely works for me.

Cool. On the [calendar](http://inside.gratipay.com/appendices/calendar).

The nice thing about Hangouts is that we get streaming and archiving on YouTube. Can we accomplish the same using Hello?

**Comment by citruspi** [comment]
@whit537,

I hadn't thought of that. I don't believe Hello offers this.

I'd be fine using Chrome for Hangouts for this.

&mdash; @citruspi

**Comment by whit537** [comment]
Eventually we'll have enough clout to start using open tools as much as possible, but for now we tend to use best-in-class so we don't slow ourselves down unnecessarily. Let's go with Hangouts for this one.

**Comment by lyndsysimon** [comment]
This issue is deluging my inbox, and I'm trying to simplify - so I've set it to ignore. If my attention is needed again, please mention me.

**Comment by whit537** [comment]
!m lyndsysimon. An appropriate thing to do. :-)

**Comment by chrisdev** [comment]
Ok so its a hangout at 1900 GMT on sunday

**Comment by whit537** [comment]
> Ok so its a hangout at 1900 GMT on sunday

Sounds good! :-)

**Comment by whit537** [comment]
Erm ... 2300 GMT == 1900 GMT-4, right? :)

**Comment by chrisdev** [comment]
Ok 7pm on Sunday. 
Reminders.app thinks that its 3pm

**Comment by aisipos** [comment]
Sorry to be late to the party - yes I will be at Pycon and will be happy to help staff the booth. I am out of town on Sunday and can't make the hangout, but will watch the archive of it.

**Comment by whit537** [comment]
Thanks, @aisipos! It'll be good to have a third person on hand to spread the load so @chrisdev and @citruspi can make it to the talks they want to go to. :-)

**Comment by citruspi** [comment]
@chrisdev @whit537,

What's the link for the Google Hangout?

**Comment by chrisdev** [comment]
I’m just on my Mac waiting for @whit537 to call. 
You logged into Gmail?

**Comment by citruspi** [comment]
@chrisdev Yup.

**Comment by citruspi** [comment]
@whit537 I just realized you may not be aware, but the email to invite me to a hangout with would be me@mihirsingh.com

**Comment by whit537** [comment]
I flaked, I'm sorry. :-(

If you're both still online I could do the call now. Otherwise, let's reschedule.

I'm sorry. :-(

**Comment by chrisdev** [comment]
@whit537 @citruspi Sleepy! Bur next week may be too late so if you got to do it now we got to do it now

**Comment by whit537** [comment]
I'm pretty sleepy, too, and we don't have @citruspi. Can we try tomorrow or Thursday at 7pm?

**Comment by chrisdev** [comment]
Ok 7pm any day of the week should be fine for me. Try for tomorrow? maybe @aisipos may be able to participate also

**Comment by whit537** [comment]
Thanks @chrisdev. Let's try tomorrow (Monday) evening at 7pm, unless @citruspi and/or @aisipos ask for Thursday instead.

**Comment by whit537** [comment]
P.S. That's 7pm in GMT-4 (= US/Eastern).

**Comment by chrisdev** [comment]
Ok it is  10:19 pm 22:19 pm now at GMT -4  city :smile:

**Comment by citruspi** [comment]
@whit537 @chrisdev Could we do this on Thursday instead of today?

**Comment by chrisdev** [comment]
Ok Thursday is ok for me

**Comment by whit537** [comment]
Roger that, Thursday it is. Thanks team. :)

**Comment by whit537** [comment]
Ready for a call @chrisdev @citruspi? Sending link in email ...

**Comment by chrisdev** [comment]
Yep!!

**Comment by citruspi** [comment]
Absolutely!

**Comment by whit537** [comment]
Stream/archive link: https://www.youtube.com/watch?v=YlUvJCvOIWk.

**Comment by whit537** [comment]
Okay! Good call, thank you @chrisdev @citruspi. Here's the summary as I understand it:

**Swag**

  - @chrisdev to bring t-shirts
  - @lyndsysimon to bring decals
  - @whit537 to ship pennies and a banner to @citruspi in Rochester the week before the conference (address to be provided)

**Schedule**

 - Load in on Thursday at 5pm.
 - Load out on Saturday at 5pm.
 - Keep at least one person at the booth from 8am to 5pm on Friday and Saturday, two or three during [busy times](https://us.pycon.org/2015/sponsors/exhibitfloor/).

 - **Controversy** - Review [this](https://medium.com/@whit537/i-resent-you-a6beb5ce19d6), [this](http://geekfeminism.wikia.com/wiki/Gittip_crisis), and [this](https://github.com/gratipay/inside.gratipay.com/issues/118). If anyone brings any of this up, LISTEN and VALIDATE. Read Peter Sandman on [risk communication](http://www.psandman.com/index-intro.htm) and especially [hostile meetings](http://www.psandman.com/col/hostile.htm). "[Hostile stakeholders always have valid grievances. Always. Find some and validate them.](http://www.psandman.com/col/hostile.htm#no-12)"
 - **Pitch**
 - ****

**Comment by whit537** [comment]
Sorry, accidentally posted too soon. :)

**Comment by whit537** [comment]
Okay! Good call, thank you @chrisdev @citruspi. Here's the summary as I understand it:

##### Swag
- @chrisdev to bring t-shirts
- @lyndsysimon to bring decals
- @whit537 to ship pennies and a banner to @citruspi in Rochester the week before the conference (address to be provided in private email)
- I guess ship stuff back to me on Sunday/Monday?

##### Schedule
- Load in on Thursday at 5pm.
- Load out on Saturday at 5pm.
- Keep at least one person at the booth from 8am to 5pm on Friday and Saturday, two or three during [busy times](https://us.pycon.org/2015/sponsors/exhibitfloor/).

##### Content

###### Pitch
1. "Gratipay is a way to give a quarter a week to someone you love or are inspired by. Do you have your phone? Want to walk through signing up right now?"
2. "Gratipay is a place to find work." ← [Teams](https://gratipay.com/explore/teams/)
3. "Gratipay is a way to run your open-source project as an open company."

###### New Since Last Year
- Started sending emails.
- Added search.
- Added Facebook and Google+.
- Added translations.
- Renamed from Gittip to Gratipay! :-)
- Retired the leaderboards (less competitive).

###### Controversy
- Review [this](https://medium.com/@whit537/i-resent-you-a6beb5ce19d6), [this](http://geekfeminism.wikia.com/wiki/Gittip_crisis), and [this](https://github.com/gratipay/inside.gratipay.com/issues/118).
- If anyone brings any of this up, LISTEN and VALIDATE. Read Peter Sandman on [risk communication](http://www.psandman.com/index-intro.htm) and especially [hostile meetings](http://www.psandman.com/col/hostile.htm).
- "[Hostile stakeholders always have valid grievances. Always. Find some and validate them.](http://www.psandman.com/col/hostile.htm#no-12)"
- I'm not there because my dad's big retirement party is that Friday.

##### Dev Priorities
1. Homepage (https://github.com/gratipay/gratipay.com/issues/3220 is the top of this rabbit hole).
2. Email notifications related to charges (https://github.com/gratipay/gratipay.com/issues/1124, https://github.com/gratipay/gratipay.com/issues/2224).
3. Stripe migration (https://github.com/gratipay/gratipay.com/issues/3245), especially if it helps with non-U.S. payins/outs.

##### Analytics

- @whit537 to provide Piwik access to @chrisdev @citruspi.
- @citruspi to run point on analytics (how many new sign-ups?).

**Comment by aisipos** [comment]
Thanks for the summary of the meeting - due to it being 4pm PDT and during work I was unable to attend. The summary above is great, I'll also watch the recording at https://www.youtube.com/watch?v=YlUvJCvOIWk

**Comment by chrisdev** [comment]
@whit537 I recall you having a design for a proposed t-shirt. Any chance you can put your hands on it?

**Comment by chrisdev** [comment]
@whit537 @citruspi @aisipos When I saw this tweet https://twitter.com/ericholscher/status/578678317068333056  from @ericholscher
I had a crazy idea! (well I actually, I had it later during my church service :))
Why don't we don't we forget the t-shirts*  and make Gratipay at Pycon all about helping some of our  "stand out" users raise more money or even meet their funding goals.

As we on-board new people or reconnect with existing users into gratipay we encourage them to help folks like Readthedocs go from $83 to $500+ for by the end of the conference.  We should choose about 4 or 5 of these "stand out" users from the Python Community.

We could also use some of the planned t-shirt money  as  fund to provide a small tip 5-25 cents? to our users. As they join, we tip them and work on trying to get them to tip one or more of our standouts.

> Hey see, someone's tipped you already. Now get out that credit/debit card  and give a tip to WothyGraipayUser

We could even create a "getto" dashboard by using tablet screens to show the progress of our standouts  are making towards meeting their goals. Ok someone's going to suggest a raspberry-pi hack but you get the picture..

We will probably need the permission of  the "standoust" Once they opt in, they can tell their own followers about this awesome promotion organised by the wonderful folks at Gratipay. In that way we can get even more buzz going

* t-shirts are too bloody expensive, even if I sourced in T&T  and the screen printer is not going to charge anything they are going to cost  $US400+

**Comment by whit537** [comment]
Sounds good to me, @chrisdev. Definitely would work best as an intentional partnership.

@ericholscher Any interest in collaborating with @chrisdev on cross-promotion at PyCon?

**Comment by whit537** [comment]
@chrisdev @citruspi @aisipos Exhibitor discount ends tomorrow. I believe we decided not to get any extra furniture. Do we want electricity?

**Comment by chrisdev** [comment]
Yes to electricity!!

**Comment by citruspi** [comment]
:zap: :+1:

**Comment by chrisdev** [comment]
@whit537 @citruspi @aisipos Here are some other folks in the Python Community that we can promote at PyCon.  Comments/suggestions

* https://gratipay.com/bitprophet/ - Fabric
* https://gratipay.com/djangogirls/ -
* https://gratipay.com/mitsuhiko/ - Flask
* https://gratipay.com/zzzeek/ - SQLAlchemy
* https://gratipay.com/CallbackWomen/

@whit537 curious why you think that it will work best as an international partnership?

**Comment by citruspi** [comment]
@chrisdev He said "intentional" not "international" :stuck_out_tongue:

**Comment by citruspi** [comment]
@chrisdev Another member of the Python community who I think we could promote would be [Kenneth Reitz](https://gratipay.com/kennethreitz/) - he created [requests](https://github.com/kennethreitz/requests).

**Comment by chrisdev** [comment]
Wow @citruspi I guess I can't even blame the auto correct for my stupidity :smile:

**Comment by citruspi** [comment]
> t-shirts are too bloody expensive, even if I sourced in T&T and the screen printer is not going to charge anything they are going to cost $US400+

Even if we don't plan on mass purchasing t-shirts to hand out for the conference, could we do something like running a [Teespring](http://teespring.com)? I'd still like to purchase one or two for myself, and I'm sure there are other who would also.

That is, assuming I like the design. :wink: 

I'd be happy to get it started and run it if there's interest. In addition, if we're interested in having a markup on the shirts, we could also use this to fundraise. We could put the profits towards providing the small initial tip for new users. I'd prefer the markup be lower than higher, so I don't think we'd come out of it with a lot of extra money, but every bit helps.

**Comment by chrisdev** [comment]
[Teespring](http://teespring.com) sounds good. Don't they have a minimum order http://teespring.com/about. Anyway, its no risk to the seller so I think we should go ahead ASAP. The markup could be used to fund the gratipay team!  I'm pretty sure I saw a design from someone called @freshdrips? @whit537 can you dig this up?

**Comment by citruspi** [comment]
@chrisdev I'm pretty sure that I've seen campaigns with twenty or thirty shirts sold. I'd personally buy two or three and I have a feeling there's interest so I'm not worried about reaching the tipping point.

**Comment by chrisdev** [comment]
Here's an rough draft of an email that we could send out to the "standout" users. 

Greetings {{ worthy_user_or_team }}
First, we would like to thank you for all the great work that you have been doing for the Python community.
We have selected your project,  along with 4 others,  for a special promotion at the Pycon 2015. In this promotion will be actively encouraging all visitors to our booth as well as our wider user base to help you attain or surpass your funding goal by the close of Pycon 2015.
We will be providing a  Dashboard at our booth that will highlight your project and the progress been made to meeting the funding target.
Thanks for participating in this initiative.

Regards {{ senior_marketing_guru }}

@citruspi @whit537 @aisipos comments and suggestions.

**Comment by citruspi** [comment]
@chrisdev @whit537 @aisipos Here's my take on it:

---

Hey {{ user }},

Gratipay appreciates the awesome work you do for the Python community. 

We're planning on having a special promotion at PyCon 2015 to help highlight
projects and teams who do awesome work and to help them get more funding
using Gratipay.

We've selected five different people and teams to take part in the promotion, and
you're one of them!

As part of the promotion, we will be encouraging people who visit the Gratipay
booth to give to people and teams who do awesome, specifically you.
We will also provide a dashboard at our booth which will highlight your project and
and the progress that has been made towards achieving your funding goal.

By the end of the conference we would like to make sure that you have reached 100%
of your funding goal.

Let us know if you're interested in partnering with us in the promotion.

\- Gratipay

**Comment by chrisdev** [comment]
@citruspi :+1:  A vast improvement!
>By the end of the conference we would like to make sure that you have reached 100%
of your funding goal.

In a few cases the user wants to raise $2,000 a week so this may be a big ask!

**Comment by lyndsysimon** [comment]
+1 to Teespring.

**Comment by whit537** [comment]
@chrisdev @citruspi I like the way this is going! A couple thoughts:

First, let's reach out to teams instead of individuals, to avoid stirring up the [resentment](https://medium.com/gratipay-blog/resentment-2c621dbd7541) we've seen in the past, and, conversely, to make the pie bigger for everyone! :-)

Second, let's not pitch this as "we've selected" but rather as "we'd love to work with you." I don't know how many projects are going to be willing to be identified with us. We should reach out to a fair number in the hopes of finding a few that are interested in this.

**Comment by chrisdev** [comment]
I afraid that most of them will say yes! 

Too many people would mean less impact. We want to show that Gratipay can work if you promote yourself.  This is what Ashe Dryen and co, did so effectively.
If this thing works we could always repeat for various events and times etc. with a different set of users/teams.

**Comment by whit537** [comment]
> I afraid that most of them will say yes!

I'm not so sure. :-(

> Too many people would mean less impact.

Not necessarily. We want to grow the whole pie. More options means more likelihood that I as a potential giver standing at the booth spot a project that I really do want to support.

> We want to show that Gratipay can work if you promote yourself.

Agreed, and I think we want *many* projects to be promoting themselves and their Gratipay account.

**Comment by chrisdev** [comment]
@whit537 you may be correct. Although I think that Gratipay may be much more positively perceived than you think.
I guess we could prepare a nice poster to show the users the list of participants.

**Comment by citruspi** [comment]
@whit537 @chrisdev

Another thought. Instead of

- purchasing $400 worth of shirts to give away or
- running a limited Teespring

we could purchase a smaller amount of shirts, about $200 or so, and sell them. The Teepsring shirts I was looking at were a little under $9/shirt for American Apparel shirts. We could either sell the shirts without a markup simply to provide people with an opportunity to purchase them or we could sell them with a slight markup to fundraise.

Either way, people at PyCon would get Gratipay shirts without us losing money on them.

I remember PyLadies was selling shirts for $20 last year. I'm wearing mine right now. :wink:

**Comment by chrisdev** [comment]
@citruspi $20 is a good price but we have to be clear what we are using the money for.
Still think we should do the Teespirng for those not at Pycon.

**Comment by whit537** [comment]
@citruspi Fine by me. My inclination would be to sell them at cost, since that's been our general pattern so far.

**Comment by citruspi** [comment]
@chrisdev To clarify, I'd set up a Teespring and then one of us would purchase some amount of shirts to sell at PyCon. That would allow us to get the PyCon shirts and enable people not going to PyCon to purchase shirts themselves.

@whit537 I'd agree. The shirts I was looking at were $8.XX so I'd probably sell them for $9. I mean we could sell them for $8.XX but I'm not a big fan of change. Actually, we could setup Stripe or Square so people could pay with cards.

**Comment by whit537** [comment]
I have a Square reader around here somewhere ...

**Comment by citruspi** [comment]
We should probably confirm that we can charge cards using Square in Canada. I know that they offer service in Canada, but I'm not sure if accounts based in the United States can charge cards in Canada.

I remember when I was in Montréal for PyCon last year, I tried to charge a card with the reader and it told me I couldn't because I wasn't in the United States.

**Comment by chrisdev** [comment]
@citruspi when can I purchase the the shirts to bring to PyCon :smile: 
Did we ever get the design from @whit537 ?

**Comment by whit537** [comment]
I've never done a t-shirt design. We have SVGs of the logo and wordmark on http://inside.gratipay.com/big-picture/brand/.

**Comment by citruspi** [comment]
@chrisdev @whit537 

According to the [Teespring website](http://answers.teespring.com/customer/portal/articles/1710948-shipping-time)
> Once a campaign ends, we begin the printing process.  This takes about a week.  From there, all domestic orders take approximately one (1) week to ship.  
> So your order will always arrive approximately two (2) weeks after the campaign has ended, not two (2) weeks after you place your order.

I think we have exactly two weeks until PyCon. If we asked nicely, we might be able to get them to rush the printing.

I'm not comfortable designing the shirts because I have no design skills and I also don't have a lot of free time in the next week or so.

If someone got a design together and sent it to me, I'd be happy to setup the campaign.

**Comment by chrisdev** [comment]
We could just use one of the logos. But the thing i like most on the page is 
![inside_gratipay](http://inside.gratipay.com/assets/inside-gratipay.svg)

**Comment by citruspi** [comment]
@whit537 @chrisdev 

So, small problem with the shirts. I was setting up a campaign to play around with it and I saw

> Your goal is the number of shirts you’re aiming to sell, but we’ll print your campaign as long as you sell enough to generate a profit.

So unless we talk to them about running a campaign without any profit, we won't be able to sell shirts at cost.

**Comment by chrisdev** [comment]
@citruspi can we make a small profit and use it as part of a fund for #160 ?

**Comment by citruspi** [comment]
@chrisdev Definitely, I just figured I'd mention it because we were interested in selling shirts at cost.

So the price of the shirts changes depending on our goal. How many shirts should we aim to sell? 50? 100? 200?

**Comment by citruspi** [comment]
@whit537 I just emailed you the address you should ship the pennies to.

How much do they weigh and how large is the container? I'm trying to think ahead as to how I'll carry them when I fly to Montréal.

**Comment by whit537** [comment]
@citruspi Looks like pennies weigh about [6 pounds per 1,000](http://www.kokogiak.com/megapenny/three.asp). The banner will be another ... five pounds? Dunno.

**Comment by whit537** [comment]
Here's the dropcloth I started with last time: http://www.harborfreight.com/9-ft-x-12-ft-canvas-dropcloth-38109.html. I think tables are 6ft. and maybe 3ft. high? So a 6 x 6 ft. square should work for the banner.

**Comment by whit537** [comment]
No weight given at that link that I can see though.

**Comment by citruspi** [comment]
@whit537 @chrisdev So, our goal for t-shirts?

Also, @whit537, could you give me access to the Piwik installation? I'd like to play around with it before PyCon.

**Comment by chrisdev** [comment]
@citruspi I think we should go with 20

**Comment by whit537** [comment]
> Also, @whit537, could you give me access to the Piwik installation?

We use LastPass to manage passwords. You should have an invite from them in your inbox. Once you sign up for LastPass I can share the Piwik login with you.

**Comment by whit537** [comment]
@citruspi I believe you should have access to the shared folder on LastPass now. Let me know if you can't get what you need for Piwik access. P.S. Login at https://gratipay.piwik.pro/. Can you let me know if LastPass is trying to bill you? Their UI is terrible and I can't tell what state you're in billing wise with us. :-(

**Comment by whit537** [comment]
I rolled 200 pennies tonight, that's all the ones I already had punched. I have another 100 or so yet to punch. I still need to get to the bank to get more. I should really get this in the mail to Rochester by Friday so that we can use ground shipping (cheaper). It'll be a stretch to pull off the banner by then but I'm confident I can at least deliver on the pennies.

**Comment by whit537** [comment]
I did 150 the other night and I got another 1000 today, punched maybe 200 of those so far.

**Comment by chrisdev** [comment]
@whit537 have we sorted out the electricity at the boot?

**Comment by citruspi** [comment]
@whit537 

> I believe you should have access to the shared folder on LastPass now.

I do!

> Can you let me know if LastPass is trying to bill you?

It says I have a free month of Pro because I was referred by a friend.

**Comment by whit537** [comment]
Okay! Pennies are all punched. :weary:

**Comment by whit537** [comment]
> have we sorted out the electricity at the boot?

No, I missed the early-bird deadline. Ball is in my court to get that lined up.

**Comment by whit537** [comment]
@citruspi I shipped 1,350 pennies (7 lbs). They should arrive by 10:30 AM on Wednesday the 8th. Will you be able to receive them then? If not I can call the shipping place tomorrow and upgrade to overnight for Tuesday delivery. Also, I neglected to get a tracking number, so I will call anyway to ask for that.

P.S. Cost was $25, plus $17 for beers with the guy that drove me to the shipping place at the last minute.

**Comment by citruspi** [comment]
@whit537 You may need to bump it to Tuesday delivery. The packages need to be processed so they're not always available the day they're received. And my flight to Montréal is Wednesday afternoon, so I would hate to have to have to fly out without them.

**Comment by whit537** [comment]
Missed 'em today (they closed at noon). Will call first thing Monday.

**Comment by whit537** [comment]
Called and upgraded, should arrive by 10:30 AM tomorrow (Tuesday). I need to stop over and swipe the card again, at which point I'll get a tracking number. +$20

**Comment by citruspi** [comment]
@whit537 Awesome! I'll let you know when I get it.

**Comment by whit537** [comment]
@citruspi Sent you the tracking number in private email.

**Comment by citruspi** [comment]
@chrisdev @aisipos We should figure out the schedule for PyCon.

**Comment by chrisdev** [comment]
@citruspi @aisipos let's meet up on Thursday morning at the Expo. We need to go to hall to setup the booth sometime before 6pm on Thursday.

**Comment by aisipos** [comment]
@chrisdev My flight arrives on Thursday at 3:48pm. Sadly means I probably would make it to the venue about 6pm. Is there anything I can bring to help?

Also - what should we use to keep in touch while we're in Montreal? Google chat, facebook messenger, whatsapp, or something else?

**Comment by citruspi** [comment]
@aisipos @chrisdev I don't personally use any of those services. Also, we won't have WiFi access everywhere and the cost of cellular data will be more expensive than the cost of SMS.

I'd vote for SMS with Twitter Direct Messages as a fallback.

**Comment by lyndsysimon** [comment]
I'm sure there will be lots of activity on IRC, official or not.

My flight comes in just before 1pm Thursday, which means I'll likely be at the convention center around 3pm or so. I'll drop by the booth with the decals (that I've still not made!) as soon as I get a chance.

I'm not sure yet if I'll have cell/data access or not - I'll be on IRC as much as possible though, so I'll let you know.

**Comment by whit537** [comment]
I ordered the electric. Receipt forwarded to @chrisdev @citruspi.

**Comment by whit537** [comment]
Any issues you need me for, my cell is 412 925 4220. I'll try to remember to keep my phone on. :)

**Comment by whit537** [comment]
P.S. Here's our current expense tally:

```
250.65 electric
 10.00 pennies
 45.00 pennies - shipping
 17.00 pennies - gratibeers
------
322.65
```

**Comment by whit537** [comment]
FedEx shows that the pennies arrived yesterday at 9:22 AM. Were you able to pick them up, @citruspi?

**Comment by citruspi** [comment]
@whit537 Yep, I have them.

**Comment by whit537** [comment]
> Yep, I have them.

Huzzah! :penguin:

**Comment by whit537** [comment]
Hear that, @Rdonnelly45? The pennies made it! :-)

**Comment by Rdonnelly45** [comment]
Mission accomplished! Nice

**Comment by whit537** [comment]
From @chrisdev in private email (names redacted):

>I’ve got some feedback from some users in the Python community.
>
A.
He actually uses his tips to fund someone else. He figures that the drop off he is seeing in his contributions has to do with lack of notifications. 
In particular found out that the CC of some of his main contributors expired but once he told them about this they started to contribute again. 
Also he removed his Gratipay Buttons form his blog Github repo and contributions started to fall. 
>
B.
Very interested in how  “Shanleygate” affected contributions. However, i think he is still positive about the mission of Gratipay/Chad and is hoping we could do something to rekindle momentum on the site.  Also, showing the stats is nice when we were growing but now it is a bit downer and may make people think is not a good platform for raising money.
>
C.
A bit sad that contributions have fallen, attributes it mainly to general lack of desire of folks to contribute to OSS even though they use it every day.  This attitude was also affecting his direct appeal efforts
>
D. & E.
They were largely using their tips to fund others. So when their tips fell then their contributions to others fell. Lack of notifications meant that they did not think about/remember their Gratipay Accounts for long periods. 
>
F.
Our problems were due to lack of marketing. Why do we not send emails like the other services on the goings on of the company. We need to have some way to promote key users
>
Others
I also got the impression that G. and others felt that they had overcome most of their negativity to the platform but they are still “cautious, wary, sceptical etc.” . The far bigger problem is that they feel that we are ineffective as fund raising platform largely because of poor UX and the fact that the stats show contributions are falling on the platform.
>
H.
He was fascinated about the Gratipay Open company model. He saw Gratipay as the future of organisational culture. Our problems were really a reflection that we were way ahead of the curve. He sounded that he wanted to make us a case-study in his next blog/book :)
